app/templates/frameworks/contract_variation.html

```diff
@@ -77,6 +79,35 @@
           {% endif %}
           </div>
 
+          {% if agreed_details %}
+          <div class="section-summary">
+            {{ summary.heading("Contract variation status", id="contract_variation_status") }}
+            {% call(item) summary.list_table(
+              [
+                {"key": "Agreed by", "value": ("%s<br />%s<br />%s" % (agreed_details.agreedUserName, agreed_details.agreedUserEmail, agreed_details.agreedAt|datetimeformat))|safe},
+                {"key": "Countersigned by", "value": ("%s<br />%s<br />%s" % 
+                  (
+                    variation_details.countersignerName, 
+                    variation_details.countersignerRole, 
+                    agreed_details.agreedAt|datetimeformat if variation_details.get('countersignedSameAsAgreed') else variation_details.countersignedAt|dateformat
```

Comment:
I don't think we need `countersignedSameAsAgreed` anywhere at all now, as it's not legally valid.

Reply:
@TheDoubleK removed